iroh-api/benches/get.rs

```diff
@@ -0,0 +1,122 @@
+use cid::Cid;
+use criterion::{black_box, criterion_group, criterion_main, BenchmarkId, Criterion};
+use futures::StreamExt;
+use iroh_api::{Api, OutType, UnixfsConfig, UnixfsEntry};
+use iroh_metrics::config::Config as MetricsConfig;
+use iroh_resolver::resolver::{Path as IpfsPath, Resolver};
+use iroh_rpc_client::{Client, Config as RpcClientConfig};
+use iroh_rpc_types::Addr;
+use iroh_store::{Config as StoreConfig, Store};
+use iroh_unixfs::{
+    chunker::{ChunkerConfig, DEFAULT_CHUNKS_SIZE},
+    content_loader::{FullLoader, FullLoaderConfig},
+};
+use tokio::runtime::Runtime;
+
+fn get_benchmark(c: &mut Criterion) {
+    let mut group = c.benchmark_group("unixfs_get_file");
+    for file_size in [
+        1024,             //  1 KiB
+        1024 * 1024,      //  1 MiB (triggers chunking)
+        10 * 1024 * 1024, // 10 MiB (triggers chunking)
+    ]
+    .iter()
+    {
+        let value = vec![8u8; *file_size];
+        let dir = tempfile::tempdir().unwrap();
+        let path = dir.path().join(format!("{}.raw", file_size));
+        std::fs::write(&path, value).unwrap();
+
+        group.throughput(criterion::Throughput::Bytes(*file_size as u64));
```

Comment:
These `as u64` casts are a bit weird.  I'd probably do the cast on the `vec![]` call above with `file_size.try_into().unwrap()` and have the values u64 to start with.

Reply:
in practice I always casted here, for throughput, as you want `usize` in all other cases, and `try_into().unwrap()` is just very gross..

Reply:
Heh, I love `.try_into().unwrap()` because it panics instead of silently doing things that are almost certainly not intended.  Maybe I just need to learn the overflow rules properly sometime...

Reply:
Well casting as `u64` from `usize` is always safe (for now) because Rust is only supported on 64bit and less systems.

Reply:
So, I'm only discussing this out of technical curiosity.  Not because I think it's very important or a blocking issue.

Having said that, I think if `<usize> as <u64>` would be "always safe" there would be a `From<usize>` on `u64`.

Reply:
I suspect it is not there because that would mean adding a 128bit target would require a breaking change in the `From` api.